### ISSUE TITLE
Amazon Linux AMI

### ISSUE THREAD

**Issue by khurramnaseem** [viljoviitanen/setup-simple-openvpn#9]
Hi,

It install like charm on said machine , but when you restart VPN server and connect vpn from you local laptop you notice it not allow you to do web browsing and additionally the EC2 instance on private subnet also not accessible , but before restart it can be ....

**Comment by viljoviitanen** [comment]
Which version of Amazon Linux?

**Comment by khurramnaseem** [comment]
[ec2-user@ip-10-0-0-206 ~]$ cat /etc/*-release
NAME="Amazon Linux AMI"
VERSION="2014.09"
ID="amzn"
ID_LIKE="rhel fedora"
VERSION_ID="2014.09"
PRETTY_NAME="Amazon Linux AMI 2014.09"
ANSI_COLOR="0;33"
CPE_NAME="cpe:/o:amazon:linux:2014.09:ga"
HOME_URL="http://aws.amazon.com/amazon-linux-ami/"
Amazon Linux AMI release 2014.09
[ec2-user@ip-10-0-0-206 ~]$ uname -a
Linux ip-10-0-0-206 3.14.20-20.44.amzn1.x86_64 #1 SMP Mon Oct 6 22:52:46 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

**Comment by viljoviitanen** [comment]
I tried to solve this, but could not. There's something funny going on with the /etc/rc.local file, it does not work as it should on Amazon Linux 2014.09, and I can't figure it out. I'll leave the issue open.

**Comment by khurramnaseem** [comment]
I suspect same that there is some thing wrong with rc.local , let me open
ticket with AWS

**Comment by MadScripts** [comment]
Can we manually execute the rc.local after reboot?

**Comment by viljoviitanen** [comment]
@MadScripts yep, sure. It works then. As a workaround I suggest to use Ubuntu 14.04 image, it works perfectly.
